app/controllers/admin/dashboard_controller.rb

```diff
@@ -0,0 +1,12 @@
+# frozen_string_literal: true
+
+module Admin
+  class DashboardController < ApplicationController
+    def index
+      authorize! :manage, :all
+      @articles = Article.where(deleted_at: nil).order(created_at: :desc)
+      @subscription_offers = SubscriptionOffer.where(deleted_at: nil).order(created_at: :desc)
+      @payment_methods = PaymentMethod.where(deleted_at: nil).order(created_at: :desc)
```

Comment:
nit: we could move the logic of filtering deleted records to the models and use [scopes](https://guides.rubyonrails.org/active_record_querying.html#scopes), maybe use a default scope?